plugins/grpc/swift-protobuf/v1.0.0/buf.plugin.yaml

```diff
@@ -0,0 +1,23 @@
+version: v1
+name: buf.build/grpc/swift-protobuf
+plugin_version: v1.0.0
+source_url: https://github.com/grpc/grpc-swift-protobuf
+description: Generates Swift client and server stubs for the gRPC framework.
```

Comment:
Do we want an integration_guide_url? Something like: https://swiftpackageindex.com/grpc/grpc-swift/2.0.0/documentation/grpccore

Reply:
works for me, https://github.com/bufbuild/plugins/pull/1723/commits/71e437ea3297b90cde4cea6c73276b932a0d1835